server/src/controllers/Events/resolver.ts

```diff
@@ -273,17 +273,62 @@ ${venue.postal_code} <br>
   }
 
   @Mutation(() => Boolean)
-  async sendEventInvite(@Arg('id', () => Int) id: number) {
+  async sendEventInvite(
+    @Arg('id', () => Int) id: number,
+    @Arg('emailGroups', () => [String], {
+      nullable: true,
+      defaultValue: ['interested'],
+    })
+    emailGroups: Array<string>,
```

Comment:
```suggestion
    emailGroups: Array<'confirmed' | 'on_waitlist' | 'canceled'>,
```
I'm sure there's a smarter way to type this, but I'm not seeing it right now.  Regardless, since we know there are only three possibilities, we can add a bit of type safety here.

Reply:
Updated the line in resolver.ts to

`    emailGroups: Array<'confirmed' | 'on_waitlist' | 'canceled' | 'interested'>,`